website/versioned_docs/version-0.4.x/set-up-your-environment.mdx

```diff
@@ -484,14 +501,142 @@ IMPORTANT: Ceramic API running on 0.0.0.0:7007
 
 By this point you should have your development environment and all configurations in place to get started working on your application.
 
+## Frequently Asked Questions
+
+---
+
+Some questions and issues come up more often than others. We've compiled a list of the most common ones here.
+
+<details>
+  <summary>Which setup method is better: wheel or JavaScript package managers?</summary>
+  <div>
+    <div>
+      Wheel is the recommended and the easiest way to configure your working environment and install
+      all the necessary dependencies. We highly recommended going with wheel if you are just
+      starting out with Ceramic. Everything will be taken care of for you.
```

Comment:
Maybe it's worth adding a sentence about why/when someone would choose JavaScript option? More manual configuration in case a project needs it, etc.

Reply:
Added.